ITKImageProcessingFilters/ImportAxioVisionV4Montage.cpp

```diff
@@ -595,7 +643,11 @@ void ImportAxioVisionV4Montage::generateCache(QDomElement& root)
 
   QString montageInfo;
   QTextStream ss(&montageInfo);
-  ss << "Columns=" << m_ColumnCount << "  Rows=" << m_RowCount << "  Num. Images=" << imageCount;
+  ss << "Max Column: " << m_ColumnCount - 1 << "  Max Row: " << m_RowCount - 1 << "  Image Count: " << imageCount;
```

Comment:
More information added to the Montage Information string which helps the user make the proper decisions based on the max rows/columns in the montage configuration file.